proxysql/test-proxysql

```diff
@@ -32,28 +32,41 @@ function process_proxysql_output {
 
 WORKDIR_ABS=$(cd ${1:-./build/proxysql}; pwd -P)
 rm -fr ${WORKDIR_ABS}/WORKDIR ${WORKDIR_ABS}/PXB ${WORKDIR_ABS}/PXC
-mkdir -p ${WORKDIR_ABS}/WORKDIR/PROXY ${WORKDIR_ABS}/PXB ${WORKDIR_ABS}/PXC
+mkdir -p ${WORKDIR_ABS}/WORKDIR/PROXY  ${WORKDIR_ABS}/WORKDIR/PXC
 
-tar -C ${WORKDIR_ABS}/PXB --strip-components=1 -zxpf $(ls $WORKDIR_ABS/pxb24.tar.gz | head -1)
-tar -C ${WORKDIR_ABS}/PXC --strip-components=1 -zxpf $(ls $WORKDIR_ABS/pxc57.tar.gz | head -1)
-tar -C ${WORKDIR_ABS}/WORKDIR/PROXY  --strip-components=1 -zxpf $(ls $WORKDIR_ABS/proxysql*.tar.gz | head -1)
+PXC_VERSION="5.7"
```

Comment:
Is it needed?

Reply:
Yes needed, but PXC_VERSION changed as paramater.